python.d/neo.chart.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Description:  netdata python.d module for monitoring Neo4j
+# Author: Jerome Baton, copied from the example of Pawel Krupa (paulfantom)
+
+
+# if you want to add a chart:
+#     insert it in ORDER
+#     copy paste and rename a chart in CHARTS
+#     code your query to return two results per line (label , value)
+
+
+# Warning
+#    charts take time to appear in the browser, also dont forget to clean the cache
+#    see /var/log/netdata/error.log to get some info and know if your code is running
+#    (python.d exited with -1  is bad news)
+
+
+from bases.FrameworkServices.SimpleService import SimpleService
+from neo4j.v1 import GraphDatabase
```

Comment:
Two things here:

* Make sure this dependency is documented when you add plugin info to the README.md file.
* This needs to fail safely so you get a reasonable error message.  Take a look at how the beanstalkd plugin (https://github.com/firehol/netdata/blob/master/python.d/beanstalk.chart.py) handles it's external dependencies, it wraps the import in a try block and sets a global variable based on the result (success or failure), and then checks that global variable in it's `check()` method and spits out an error message there if the import failed.